src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();

```

Comment:
I would disagree. Because the essence of what we do is to rename the nested Interface IFace, and for this we only need a name.

Otherwise, the semantics is unclear. Suppose we have `spoon.test.template.testclasses.InvocationTemplate.IFace` and we want to replace by `a.b.c.Foo`: should the result be `spoon.test.template.testclasses.InvocationTemplate.Foo` (current behavior) or `spoon.test.template.testclasses.InvocationTemplate.a.b.c.Foo`?

AFAIK, the package is information is useless.